core/domain/collection_services.py

```diff
@@ -631,6 +631,9 @@ def apply_change_list(collection_id, change_list):
 
     Returns:
         Collection. The resulting collection domain object.
+
+    Raises:
+        Exception. The change_list is not applicable on given collection_id.
```

Comment:
```suggestion
        Exception. The change list is not applicable on the given collection.
```

In general I think you don't need to reference specific variables in these messages. Please try to fix it elsewhere.

Reply:
I tried correcting this mistake. 
Thanks!!!